custom-collections.go

```diff
@@ -0,0 +1,67 @@
+package shopify
+
+import "time"
+
+// CustomCollections is a list of CustomCollections.
+type CustomCollections []CustomCollection
+
+// CustomCollection is a grouping of products that a merchant can create to make their store easier to browse.
+type CustomCollection struct {
+	// BodyHTML is the description of the custom collection. Includes HTML markup. Many shop themes display this on the custom collection page.
+	BodyHTML string
+	// Handle is a human-friendly unique string for the custom collection. Automatically generated from the title. Used in shop themes by the Liquid templating language to refer to the custom collection. (maximum: 255 characters) */
+	Handle string
+	// ID is the ID for the custom collection.
+	ID int64
+	// Image is the image associated with the custom collection.
+	Image Image
+	// PublishedAt is the time and date when the custom collection was made visible. Returns 0 for a hidden collection.
+	PublishedAt time.Time
+	// PublishedScope is whether the custom collection is published to the Point of Sale channel.
+	/*
+		Valid Values:
+		- web: The collection is published to the Online Store channel but not published to the Point of Sale channel.
+		- global: The collection is published to both the Online Store channel and the Point of Sale channel.
+	*/
+	PublishedScope string
+	// Rules is the list of rules that define what products go into the custom collection.
+	Rules Rules
```

Comment:
I think rules are only on smart collections, this is a custom collection which doesnt have them

Reply:
ah forgot about that. I have removed it.